applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -651,6 +692,17 @@ static bool event_handler(const struct event_header *eh)
 
 		k_free(item);
 
+		if (IS_ENABLED(CONFIG_DESKTOP_HID_FORWARD_LOW_LATENCY_ON_HID_INPUT)) {
+			const struct hid_report_sent_event *event =
+				cast_hid_report_sent_event(eh);
+			u8_t sub_idx = rep_id_2_sub_idx[event->report_id];
+
+			if (sub_idx != INVALID_SUB_IDX &&
+			    subscribers[sub_idx].llpm_support) {
+				switch_to_low_latency(&subscribers[sub_idx]);
```

Comment:
I don't think it should be here. Why not handling this when report is received from BLE?